pandas/tests/series/methods/test_nlargest.py

```diff
@@ -211,3 +211,14 @@ def test_nlargest_boolean(self, data, expected):
         result = ser.nlargest(1)
         expected = Series(expected)
         tm.assert_series_equal(result, expected)
+
+    def test_nlargest_nullable(self, any_nullable_numeric_dtype):
+        # GH#42816 NB so far we only handles cases without NAs
```

Comment:
hmm, so this PR does not close the issue since this worked in 1.2.5?

Reply:
comment is out of date, this should now handle cases with NAs, though a test case needs to be added